src/app/public/modules/country-field/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './country-field.component';
+export * from './country-field.module';
+export * from './types';
```

Comment:
To prepare for Angular 9, it would be better to delete this index.ts file and put all exports on the src/app/public/index.ts file.

Reply:
Done. I think I'm exporting thing correctly now. Please let me know if not